pantheon-bundle/src/main/resources/SLING-INF/nodetypes/nodetypes.cnd

```diff
@@ -57,6 +57,8 @@
 [pant:symlink] > nt:unstructured
     - sling:resourceType (string) = 'pantheon/symlink' mandatory autocreated
 
+[pant:bucket] > nt:unstructured, mix:created, mix:lastModified
+    - sling:resourceType (string) = 'pantheon/bucket' mandatory autocreated
```

Comment:
I asked earlier in our QA call but I'll ask here so it's recorded - why is creating this new type necessary?

Reply:
The node defines with this new type has a special purpose. for instance, it's a repository level node. It may contain a standaloneAttributeFile for the repository.